src/blocks/modal.css

```diff
@@ -139,13 +147,51 @@
   cursor: pointer;
 }
 
+.modal__delete-btn {
+  background-color: #ff6565;
+}
+
+.modal__input_type_error {
+  border: 1px solid rgba(255, 0, 0, 0.6);
+}
+
 .modal__submit-btn:hover {
   opacity: 70%;
 }
 
 .modal__submit-btn_disabled {
   background-color: #212121;
-  opacity: 50%;
+}
```

Comment:
* The **submit** button should become `gray` if the form is `invalid`


The `opacity` is needed here because it makes the button `gray` and look `disabled`